common/configurable/src/main/java/io/helidon/common/configurable/ThreadPool.java

```diff
@@ -383,6 +383,7 @@ protected void afterExecute(Runnable r, Throwable t) {
     @Override
     public void shutdown() {
         Event.write();
+        ObserverManager.unregisterExecutorService(this);
         super.shutdown();
     }
```

Comment:
What happens when `shutdownNow()` is called?

Reply:
Good point. I've added an override of `shutdownNow`.